px-tooltip.html

```diff
@@ -137,6 +138,50 @@
         type: String,
         value: ''
       },
+
+      /**
+      * This boolean can be used to REQUEST an open/close action to the tooltip
+      * Because the show/hide mechanism are asynchronous this won't represent
+      * the actual presence of the tooltip on the screen, this is represented
+      * by the `visible` property
+      */
+      opened: {
+        type: Boolean,
+        value: false,
+        notify: true
+      },
+
+      /**
+        * Whether the tooltip is currently shown on screen
+        *
+        */
+      visible: {
+        type: Boolean,
+        value: false,
+        readOnly: true,
+        notify: true
+      },
+      /**
+        * If true the tooltip will ignore the targets events it usually listens
+        * to: mouseup/down, mouseleave/enter... Use this to more precisely
+        * manually control the showing/hiding of the tooltip
+        */
+      ignoreTargetEvents: {
+        type: Boolean,
+        value: false
+      },
+      /**
+        * True when opened has been requested (i.e set to true) but the tooltip
+        * hasn't been made visible yet. Not needed in most cases but might be
+        * helpful to catch edge states (i.e reclosing the tooltip after a show
+        * request has been issued but the tooltip isn't visible yet)
+        */
+      requestShow: {
```

Comment:
Nitpicky but maybe `showRequested` or `openRequested`? Since this sounds like a method to request it to be shown.

Reply:
👍  went for openRequested to make it more obvious it link to `opened`